lib/src/fnc/util/math/nearestrank.rs

```diff
@@ -1,11 +1,15 @@
-use crate::sql::number::Number;
+use super::percentile::Percentile;
+use crate::sql::number::{Number, Sorted};
 
 pub trait Nearestrank {
-	fn nearestrank(self, _: Number) -> Number;
+	/// (Assuming this is an alias for Percentile or is this a numeric rank)
```

Comment:
Could you look here: https://github.com/surrealdb/surrealdb/blob/golang/util/math/percentile.go

There was Percentile, and NearestRankPercentile functions, that were slightly different

Reply:
Sure thing! I'll square my assumptions with what's going on in the `golang` branch and bring them in line 😄